mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -33,9 +33,20 @@ CLASS aij net/minecraft/entity/mob/MobEntity
 	METHOD H cannotDespawn ()Z
 	METHOD I checkDespawn ()V
 	METHOD K mobTick ()V
+	METHOD a changeAngle (FFF)F
```

Comment:
What exactly is an `angle`? I am only aware of `pitch` and `yaw` (maybe `roll`, but minecraft don't use it often)

Reply:
In the code this function works on `pitch` or `yaw` independently, so the angle refers the whichever the function is called with.